### DISCUSSION TITLE
/latest/examples/wind.html :  rotation and flux directions ?

### DISCUSSION THREAD

**Discussion by jipexu** [openlayers/openlayers#16222] [Q&A] [answered]
Hello, very nice work with this new implementation! I know it's experimental. When the wind layer rotates it seems that the "direction (or the visual perception of the direction (reversed?)) of the flux" are wrong ??. Once again, congratulations because this native implementation is a big step forward !
I have to precise what happen's or(what i feel/see) : 
- centered the map on western europe : 
the wind seems to go from the west and finish rolled at the south of UK (is it right ?)
- rotated the map 90° to the east 
the wind seems now to go from the east ! ...

so effectively the rotation is OK but the direction not ? or it is optical probleme ?...

**Comment by kikuchan** [comment, marked as answer]
Thank you for reporting.
You're right, the direction is surely opposite.

I've just filed a PR for this issue, and it will be resolved in time.

**Reply by jipexu** [reply]
Thanks for your answer !
